test/lib/wait-until-called.js

```diff
@@ -1,22 +1,41 @@
+const TEN_SECONDS = 10000
```

Comment:
We should probably rename this to something like `CALL_TIMEOUT`

Reply:
:sweat_smile: Yes, definitely. Good idea

Reply:
I've renamed it to `DEFAULT_TIMEOUT`